src/media/css/category-icons.styl

```diff
@@ -1,33 +1,35 @@
+/*
+    Stylesheet for category icon using the spritesheet.
+    Defines the classes for each individual icon as well as its applications
+    within pages.
+*/
 @import 'lib';
 
 $cat-icon-size-default = 28px;
-$cat-icon-size-desktop = 60px;
 
-// Mixin to handle hover/focus states of svg sprite.
-// One day if background-position-x/y works we can
-// factor a lot of this out.
 sprite-background-position($vertical-pos) {
+    // Handles which color sprite to show on which pages.
     // White.
-    &.cat-hovercat:hover:before,
-    &.mobile-cat-header:before {
+    &.cat-hovercat:hover:before {
+        // Icons in category drop-down menu when hovered.
         background-position: 100% $vertical-pos;
     }
 
     // Blue.
-    // The icons in category drop-down menu.
-    // The icon in desktop header on cat landing.
-    // The icons on `/categories` index.
-    &.cat-hovercat:before,
     &:before,
-    &.desktop-cat-header:before {
+    &.cat-hovercat:before,
+    &.category-header-icon:before {
+        // The icons in category drop-down menu.
         background-position: 0 $vertical-pos;
     }
 
-    // Gray.
-    // The icon in mobile header on cat landing.
-    &.mobile-cat-header:before {
-        background-position: 50% $vertical-pos;
+    @media $base-tablet {
+        &.category-header-icon:before {
+            // 75px icon in category header, which needs to be cenetered.
+            background-position: 107.5% $vertical-pos - 0.7;
```

Comment:
Not sure if the `- 0.7` is quite right since it looks better for me without it:

![](https://s3.amazonaws.com/f.cl.ly/items/193z0t3T39203d451Z02/Image%202015-01-27%20at%202.39.59%20PM.png)

vs the current:

![](https://s3.amazonaws.com/f.cl.ly/items/1J3L2E1z102j0g1v0Q2R/Image%202015-01-27%20at%202.39.11%20PM.png)

@pwalm ?